### PR TITLE
Snowflake Native App: Add feature parity warning and rename app in  section

### PR DESCRIPTION
**Pull request by maxi297**
## What
Add warning to documentation for feature scope

### CODE REVIEW

**Comment by maxi297** [comment]
@katmarkham do you want me to update the name of the section [here](https://github.com/airbytehq/airbyte/blob/6340436a767acac623ced7e9a240082342d18348/docs/snowflake-native-apps/linkedin-ads.md#snowflake-native-app-authorizations) as well?

File: docs/snowflake-native-apps/linkedin-ads.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The LinkedIn Ads Connector by Airbyte is a Snowflake Native Application that allows you to extract data from your LinkedIn Ads account and load records into a Snowflake database of your choice.
 
 :::info
-The LinkedIn Ads Connector is in _private preview_ and is subject to further development that may affect setup and configuration of the application.
+The LinkedIn Ads Connector is in _private preview_ and is subject to further development that may affect setup and configuration of the application. It also does not have feature-parity with the Airbyte platform as only full refresh without dedup is supported.
```
Comment (katmarkham) [inline docs/snowflake-native-apps/linkedin-ads.md:6]:
```suggestion
The LinkedIn Ads Connector by Airbyte is in _private preview_ and is subject to further development that may affect setup and configuration of the application. Please note that, at this time, only a full table refresh without dedupe is supported.  
```

**Review by katmarkham — COMMENTED** [review]
thanks for updating

**Review by katmarkham — APPROVED** [review]
looks good

**Pull request merged**